apps/dashboard/app/models/batch_connect/app.rb

```diff
@@ -216,12 +216,28 @@ def build_session_context
     # @param session_context [SessionContext] object with attributes
     # @param fmt [String, nil] formatting used for attributes in submit hash
     # @return [Hash] hash of submission options
-    def submit_opts(session_context, fmt: nil)
+    def submit_opts(session_context, staged_root: staged_root, fmt: nil)
```

Comment:
```suggestion
    def submit_opts(session_context, staged_root: nil, fmt: nil)
```
Get circular issues with doing `staged_root: staged_root`